Addition of Recharge Node CVPM 08

mlmaskey opened the pull request:
One recharge node RECH8 is added and connected with c173 and hgr08 via two links: c173-rech8 and rech8-hgr08

mlmaskey committed:
Addition of Recharge Node CVPM 08

One recharge node RECH8 is added and connected with c173 and hgr08 via two links: c173-rech8 and rech8-hgr08

mlmaskey commented:
@nickrsan and @msdogan 
I created new branch with a recharge node near CVPM8. Please let me know if it works.
Thanks

mlmaskey commented:
Hi @msdogan, 
As I mentioned in the meeting, the flow at the recharge node is not appearing. Could you please let me know what is the implication. In fact, I am running a full model for 82 years. Ther results are not completed yet. But the situation does not show any improvement. Please let me know if we can discuss.

mlmaskey committed:
Update README.md

nickrsan commented:
Hi @msdogan, in talking with Mahesh, I'd expressed that I didn't think we should add this node to the core calvin network data until it's an active facility in California and right now it's proposed facility, but we wanted your input. I'd rather wait than need to deal with removing it if the facility ends up not existing.

msdogan commented:
Hi @nickrsan, that's a good idea. @mlmaskey please work on a separate branch and once you finish all your changes and then we can decide to merge onto master. We usually do not merge this hypothetical scenarios. I would work on a different branch for your project.

msdogan commented:
@mlmaskey your file is incorrect here: 

data/rech8-hgr08/terminus

mlmaskey commented:
> 
> 
> @mlmaskey your file is incorrect here:
> 
> data/rech8-hgr08/terminus
@msdogan, how it could be. I do not see the location of file is wrong. Could you please point out the error is...
../san-joaquin-river/eastern-valley-floor/hgr08

msdogan commented:
@mlmaskey  if all file paths are correct, maybe check on upper and lower cases and make sure they are consistent.

mlmaskey commented:
> 
> 
> @mlmaskey if all file paths are correct, maybe check on upper and lower cases and make sure they are consistent.
@msdogan I am not sure that all the letters are not consistent. But it does runs without giving not change in the results even I added recharge node. 
Thanks

mlmaskey committed:
Cost Changed

Recharge cost 1500/12: annualized cost, Figure 4
Perrone, D., & Rohde, M. M. (2016). Benefits and economic costs of managed aquifer recharge in California. San Francisco Estuary and Watershed Science, 14(2).
